generatedocs.py

```diff
@@ -0,0 +1,413 @@
+from git import Repo # git for downloading assets
```

Comment:
Getting:
`No module named 'git'`
when running `make html`.  The requirements.txt mentions GitPython instead of Git, so maybe it is missing the actual dependency?

Reply:
Are you running the python virtual environment with `.\.venv\Scripts\activate` ?

Reply:
I have updated the Readme with the proper way to run it

Reply:
Works great now!